lib/route-recognizer.js

```diff
@@ -125,10 +125,7 @@ function parse(route, names, specificity, shouldDecodes) {
   // `x`, irrespective of the other parts.
   // Because of this similarity, we assign each type of segment a number value written as a
   // string. We can find the specificity of compound routes by concatenating these strings
-  // together, from left to right. After we have looped through all of the segments,
-  // we convert the string to a number.
-  specificity.val = '';
```

Comment:
Moved to https://github.com/tildeio/route-recognizer/pull/103/files#diff-3cd8d4dfb826ed2ebdceb9c43e729d8eR335